pkg/scaffold/internal/templates/v2/crd/enablecainjection_patch.go

```diff
@@ -42,7 +42,7 @@ func (f *EnableCAInjectionPatch) SetTemplateDefaults() error {
 			fmt.Sprintf("cainjection_in_%s.yaml", plural))
 	}
 
-	f.TemplateBody = EnableCAInjectionPatchTemplate
+	f.TemplateBody = enableCAInjectionPatchTemplate
```

Comment:
I am curious why the lint did not get this.

Reply:
Same here, maybe `golint` is being launched with different flags or something.